package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "clownface",
-  "version": "1.4.0",
+  "version": "1.4.1",
```

Comment:
Please revert. You shouldn't bump the version in a PR but only when actually releasing